docs/features/secret-store/provider/key-vault.md

```diff
@@ -1,6 +1,7 @@
 ---
 title: "Azure Key Vault secret provider"
 layout: default
+redirect_from: /features/auth/azure-key-vault
```

Comment:
According to the supported Jekyll plugins on GitHub and the [docs](https://www.rubydoc.info/gems/jekyll-redirect-from/0.16.0) on the `jekyll-redirect-from` plugin, all I have to do is add it here. But the deploy preview is in a infinite loop of redirecting, so I can't verify if this is working. And I'm not sure if this single line is the reason it's in this loop also.

Reply:
That's ok, let's merge. I don't think the preview supports this.

Reply:
Feel free to merge and PR in case it doesn't work

Reply:
Hmm, I don't think it's working 😢